library/src/main/java/com/google/android/material/motion/runtime/Scheduler.java

```diff
@@ -172,6 +172,38 @@ public void addPlan(Plan plan, Object target) {
     getTargetScope(target).commitPlan(planInfo);
   }
 
+  /**
+   * Adds a {@link NamedPlan} to this scheduler.
```

Comment:
This should be expanded on in the latest commit. github doesn't seem to be picking up on that though? Happy to make additional changes here though if you think my description of what actually happens need some more work.